### ISSUE TITLE
Rotation triggers only in initial rectangle

### ISSUE THREAD

**Issue by kirillt** [ARK-Builders/ARK-Retouch#45]
See on the picture:
1. Blue zone — initial bounds of the image, before rotation.
2. Orange zone — current bounds, after rotation.
3. Unpainted zone — intersection of initial and current bounds.

![bug-rotation.png](https://user-images.githubusercontent.com/581023/236373895-9f809682-2b70-487e-bf16-5a0382477d85.png)

So, in rotation dialog, if user made 1 rotation and want to do new rotation without leaving the dialog, it is triggered only by touching the screen in initial bounds of the image (blue + unpainted zone). Touching the screen in orange zone doesn't trigger rotation. This is counter-intuitive for an average user.

In other words, only initial rectangle is draggable, not current one.
